lint-config.json

```diff
@@ -1,6 +1,6 @@
 {
   "Exclude": ["src/terraform/bindata.go"],
-  "Disable": ["gotype", "gotypex"],
+  "Disable": ["gas", "gotype", "gotypex"],
```

Comment:
There were a lot of errors in the lint output for this as `gas` was unable to load types (same problem gotype was hitting)